environment.yml

```diff
@@ -19,11 +20,16 @@ dependencies:
   - pydantic
   - pymbar
   - python >= 3.6
+  - pymol
```

Comment:
Wow, do we need pymol?!

Reply:
It looks like we don't use it---should we remove this and the `schrodinger` channel?

Reply:
I'll double check, if we don't need it I'll remove it before merging

Reply:
Removed